apispec/ext/flask.py

```diff
@@ -72,6 +111,20 @@ def path_from_view(spec, view, **kwargs):
     path = Path(path=path, operations=operations)
     return path
 
+def path_from_method_view(spec, method_view, **kwargs):
+    """Path helper that allows passing a Flask MethodView view function."""
+    path = path_from_view(spec, method_view)
+    operations = {}
+    for method in method_view.methods:
+        method_name = method.lower()
+        view = getattr(method_view.view_class, method_name)
+        docstring_yaml = utils.load_yaml_from_docstring(view.__doc__)
+        operations[method_name] = docstring_yaml or dict()
+
+    path.operations.update(operations)
+    return path
+
 def setup(spec):
     """Setup for the plugin."""
     spec.register_path_helper(path_from_view)
+    spec.register_path_helper(path_from_method_view)
```

Comment:
Is it necessary to register multiple path helper functions in one extension? I think this breaks the locality of the conditional branch whether the view is derived from MethodView or not.